flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/plan/nodes/logical/FlinkLogicalTableFunctionScan.scala

```diff
@@ -96,11 +96,12 @@ class FlinkLogicalTableFunctionScanConverter
       return false
     }
     val rexCall = logicalTableFunction.getCall.asInstanceOf[RexCall]
-    if (!rexCall.getOperator.isInstanceOf[TableSqlFunction]) {
-      return false
+    val functionDefinition = rexCall.getOperator match {
+      case tsf: TableSqlFunction => tsf.udtf
+      case bsf: BridgingSqlFunction => bsf.getDefinition
```

Comment:
This is just for my understanding, but where is the bridging function created?

Reply:
In the catalog layer, see `FunctionCatalogOperatorTable`.